app/policies/user_policy.rb

```diff
@@ -0,0 +1,38 @@
+class UserPolicy < ApplicationPolicy
+  attr_reader :current_user, :user
+
+  def initialize(current_user, user)
+    @current_user = current_user
+    @user = user
+  end
+
+  def create?
+    true
+  end
+
+  def edit?
+    user == current_user
+  end
+
+  def update?
+    edit?
+  end
+
+  def destroy?
+    edit?
+  end
+
+  def show?
```

Comment:
almost! Is your app displaying properly for users who are private. Try moving this logic to a method called show_photos? and setting show to true. Call show_photos? policy in your users#show template